CustomerData/CustomSection.php

```diff
@@ -0,0 +1,145 @@
+<?php
+namespace Balancepay\Balancepay\CustomerData;
+
+use Balancepay\Balancepay\Helper\Data as BalancepayHelper;
+use Balancepay\Balancepay\Model\Config as BalancepayConfig;
+use Balancepay\Balancepay\Model\Request\Factory as RequestFactory;
+use Magento\Customer\Api\CustomerRepositoryInterface;
+use Magento\Customer\CustomerData\SectionSourceInterface;
+use Magento\Customer\Model\Session;
+use Magento\Framework\Pricing\Helper\Data;
+
+/**
+ * Class CustomSection
+ * @package Balancepay\Balancepay\CustomerData
+ */
+class CustomSection implements SectionSourceInterface
```

Comment:
What is this section? can you come up with more descriptive class name?

Reply:
Updated